.github/ISSUE_TEMPLATE/issue_report.yml

```diff
@@ -0,0 +1,106 @@
+name: Issue Report
```

Comment:
Why don't we name this Bug Report?

Reply:
My idea was to keep it more general. Currently there is a bug report template in Fast DDS repository but most users choose to open a blank issue instead. Even though we understand an issue mostly as a bug, some users may be confused if when opening an issue they can only pick the bug report and they do not think that the issue is a proper bug. That said, I am not against renaming it to Bug report so the user thinks twice about opening the issue.
@EduPonz what do you think?

Reply:
Renaming to Bug report